### PR TITLE
[BTAT-5831] - hide business name and remove option to change ppob for overseas businesses

### PR DESCRIPTION
**Pull request by sraspin**

### CODE REVIEW

**Comment by sraspin** [comment]
Don't merge until BTAT-5827 for vat-subscription has been merged!

File: app/views/customerInfo/customer_circumstance_details.scala.html [hunk 1]
```diff
@@ -79,22 +79,24 @@
         <dd id="businessAddress" class="cya-answer">
             @render_address(address)
         </dd>
-        <dd class="cya-change">
+        @if(!circumstances.customerDetails.overseasIndicator) {
```
Comment (TheDeadlyPianist) [inline app/views/customerInfo/customer_circumstance_details.scala.html]: This seems to hide the PPOB if overseas. The story says that the PPOB should be shown, and only the change button should be hidden.

Comment (sraspin) [reply app/views/customerInfo/customer_circumstance_details.scala.html]: This does only hide the change link doesn't it?

**Comment by TheDeadlyPianist** [comment]
The functionality for this is going to need feature switching. Sorry. :(

File: app/connectors/httpParsers/CustomerCircumstancesHttpParser.scala [hunk 2]
```diff
@@ -16,24 +16,27 @@
...
+import play.api.{Configuration, Logger, Play}
 import play.api.http.Status
 import uk.gov.hmrc.http.{HttpReads, HttpResponse}
 
-
 object CustomerCircumstancesHttpParser {
 
   implicit object CustomerCircumstanceReads extends HttpReads[HttpGetResult[CircumstanceDetails]] {
 
+    implicit val config: Configuration = Play.current.configuration
```
Comment (TomDudley92) [inline app/connectors/httpParsers/CustomerCircumstancesHttpParser.scala]: Could there be a cleaner way to get this as this way is deprecated 😿

File: app/testOnly/views/featureSwitch.scala.html [hunk 3]
```diff
@@ -42,6 +42,7 @@
               @views.html.templates.inputs.singleCheckbox(form(ConfigKeys.allowAgentBankAccountChange), "Allow Agents to change Bank Account")
               @views.html.templates.inputs.singleCheckbox(form(ConfigKeys.makingTaxDigitalSectionFeature), messages(ConfigKeys.makingTaxDigitalSectionFeature))
               @views.html.templates.inputs.singleCheckbox(form(ConfigKeys.useLanguageSelectorFeature), messages(ConfigKeys.useLanguageSelectorFeature))
+              @views.html.templates.inputs.singleCheckbox(form(ConfigKeys.useOverseasIndicator), messages(ConfigKeys.useLanguageSelectorFeature))
```
Comment (TomDudley92) [inline app/testOnly/views/featureSwitch.scala.html]: Need to use the `useOverseasIndicator` config key for the messages not `useLanguageSelectorFeature`

Comment (TheDeadlyPianist) [reply app/testOnly/views/featureSwitch.scala.html]: Oops.

File: conf/application.conf [hunk 4]
```diff
@@ -117,6 +117,7 @@ features {
   allowAgentBankAccountChange.enabled = false
   makingTaxDigitalSection.enabled = true
   useLanguageSelector.enabled = true
+  useOverseasIndicator.enabled = true
```
[the remark this answers is not in the document]
Comment (TheDeadlyPianist) [reply conf/application.conf]: Not yet, I'll make them now.

**Pull request merged**